setup.py

```diff
@@ -17,11 +17,13 @@
       author_email='dev@uservoice.com',
       packages=['uservoice'],
       install_requires=[
+          'future',
           'pycrypto',
           'pytz',
           'PyYAML',
           'requests',
-          'future',
+          'simplejson',
```

Comment:
Instead of reintroducing the simplejson dependency -- which was [intentionally removed](https://github.com/uservoice/uservoice-python/commit/f5801903fd183455114b062bb23fcbadd6c6128b) -- could we instead fix the bug to work with the stdlib json module?

Reply:
that would definitely be a better approach but i'm not sure i will have time to make the change personally.